system-addon/content-src/styles/_theme.scss

```diff
@@ -0,0 +1,80 @@
+// Default theme
+.outer-wrapper {
+  --newtab-background-color: $grey-10;
+  --newtab-text-primary-color: $grey-90;
+  --newtab-text-secondary-color: $grey-50;
+  --newtab-text-conditional-color: $grey-60;
+  --newtab-link-primary-color: $blue-60;
+  --newtab-link-secondary-color: $teal-70;
+  --newtab-border-primary-color: $grey-40;
+  --newtab-border-secondary-color: $grey-30;
+  --newtab-icon-primary-color: $grey-90-80;
+  --newtab-icon-secondary-color: $grey-90-60;
+  --newtab-icon-tertiary-color: $grey-30;
+  --newtab-textbox-border: $grey-90-20;
+  --newtab-textbox-color: inherit;
+  --newtab-active-element-color: $grey-20-60;
+  --newtab-button-primary-color: $blue-60;
+  --newtab-button-secondary-color: inherit;
+  --newtab-contextmenu-button-color: $white;
+  --newtab-modal-color: $white;
+  --newtab-overlay-color: $grey-20;
```

Comment:
I guess up to here, the variables are pretty generic. And all the ones below are pretty specific to different sections.